hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -28,62 +28,57 @@
 public class LineageHook implements org.apache.hadoop.hive.ql.hooks.ExecuteWithHookContext {
 	private static final Logger LOGGER = Logger.getLogger(LineageHook.class);
 	private boolean successFlag;
-	public boolean isSuccess() {return successFlag;}
+	public boolean isSuccess() {
+	return successFlag;
+	}
 
+	@Override
 	public void run(HookContext hookContext) {
-//		TODO - redirect System.out to LOGGER.debug
-		System.out.println("Running LineageHook...");
-		String wholeQuery = null;
-		String processId = null;
-		String instanceExecId = null;
+	LOGGER.debug("Running LineageHook...");
+	String wholeQuery = null;
+	String processId = null;
+	String instanceExecId = null;
 
-	    try {
-		    if (hookContext == null) {
-			    LOGGER.warn("HookContext is null. Assuming default values");
-			    wholeQuery = LineageConstants.query;
-			    processId = "" + LineageConstants.processId;
-			    instanceExecId = "" + LineageConstants.instanceId;
+	try {
+	if (hookContext == null) {
+		LOGGER.warn("HookContext is null. Assuming default values");
+		wholeQuery = LineageConstants.query;
+		processId = "" + LineageConstants.processId;
+		instanceExecId = "" + LineageConstants.instanceId;
 
-		    } else {
-//			    hookContext.getConf().getAllProperties().list(System.out);
-			    wholeQuery = hookContext.getQueryPlan().getQueryString();
+	}
+	else {
+		wholeQuery = hookContext.getQueryPlan().getQueryString();
 
-			    Properties properties = hookContext.getConf().getAllProperties();
-			    for (Map.Entry entry : properties.entrySet()) {
-				    String key = (String)(entry.getKey());
-				    if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
-					    processId = (String) (entry.getValue());
-					    System.out.println("ProcessId found in Hive = " + processId);
-				    } else if (LineageConstants.instanceExecIdString.equalsIgnoreCase(key)) {
-					    instanceExecId = (String) (entry.getValue());
-					    System.out.println("InstanceExecId found in Hive = " + instanceExecId);
-				    }
-			    }
+		Properties properties = hookContext.getConf().getAllProperties();
+		for (Map.Entry entry : properties.entrySet()) {
+			String key = (String)(entry.getKey());
+			if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)